entity-framework/core/cli/services.md

```diff
@@ -34,19 +34,32 @@ The following is a list of the design-time services.
 Service                                                                              | Description
 ------------------------------------------------------------------------------------ | -----------
 <xref:Microsoft.EntityFrameworkCore.Design.IAnnotationCodeGenerator>                 | Generates the code for corresponding model annotations.
+<xref:Microsoft.EntityFrameworkCore.Design.ICandidateNamingService>                  | Generates candidate names for entities and properties.
```

Comment:
No objection to updating this, but I'm a bit dubious about the whole section - this is all very internal stuff, which seems useful only for writers of providers (or very specific kinds of tools); we don't have this kind of internal info for anything else outside of design, so I'm not sure why it makes sense specifically for design.

Maybe we should move it to https://learn.microsoft.com/en-us/ef/core/providers/writing-a-provider or similar?